README.md

```diff
@@ -24,6 +27,84 @@ The Xcode project is configured to use 4 spaces for indentation.
+
+#### Clean Architecture layers
+
+- _UI_ with SwiftUI
+- _Presentation_ with the ViewModel
+- _Domain_ for domain models, UseCases and other domain logic.
+- _Data_ for data storage and retrieval.
+
+#### Use cases
+
+- Use cases are single-purpose: GetUserUseCase, but also: GetUserWithArticlesUseCase.
+- Use cases can call other use-cases.
+- Use cases do not have state, state preferably lives in the data layer.
+
+#### Dependency injection
+
+Dependency injection (DI) means that objects don't instantiate the objects or configurations that they require themselves, but they are passed in from the outside.
+This is useful, because it makes code easier to test and easier to change. It promotes good separation of concerns.
+
+We use [Factory](https://github.com/hmlongco/Factory) as a DI container.
+
+### Modules & libraries
+
+* Preferably use the Swift Package Manager for dependencies. Use other package managers only if there's no other option.
+* Only extract code into a package if there are strong reasons to do so. For example:
+    * It is used from at least two different targets/packages, or is a candidate to be extracted to an open-source package.
+    * It is completely self-contained.
+
+When choosing a third-party library, prefer libraries that:
+
+* Are written in idiomatic Swift or Objective-C that sticks to best practices.
+* Have as few dependencies of its own as possible. Preferably none.
+* Aren't too big, in order to keep compile times and bloat in check.
+
+### Testing
+
+* For business logic, we write unit tests.
+* For testing the user interface, we write UI tests in a behaviour-driven way using the [Salad](https://github.com/Q42/Salad) library.
+* Tests are run on CI (GitHub Actions). Tests must pass before a PR may be merged and before any sort of build is created.
+
+### Views
+
+* Keep views small and focused. When a view becomes large, split it up into smaller views.
+* Every view gets a UI preview if at all possible. The preview should show the view in different states using dummy data.
+* We use [custom SF Symbols](https://developer.apple.com/documentation/uikit/uiimage/creating_custom_symbol_images_for_your_app/) whenever a custom icon is needed, so that they render in a consistent manner. 
+
+### Accessibility
+
+* Every new component or control should be audited for basic accessibility support:
+    * Dynamic type size support
+    * VoiceOver support
+* Also consider:
+    * Bold text support
+    * High contrast support
+* Use `accessibilityRepresentation` on custom controls to make them accessible. 
+
+### Localization
+
+String catalogs are used to localize the project. The default languages supported are English and Dutch.
+
+### Async code
+
+* Asynchronous code should be written using async/await whenever possible.
+* [Combine](https://developer.apple.com/documentation/combine) should only be used when async/await or `AsyncSequence` fall short, and more complexity is needed to solve the problem at hand.
```

Comment:
I think I would steer away from words like `should`.  And let's add some explanation why. Something like this? 
```suggestion
* `async`/`await` is preferred over Combine/Promises/etc. to leverage the compiler concurrency checking.
* [Combine](https://developer.apple.com/documentation/combine) can be used when `async`/`await` or `AsyncSequence` fall short, and more complexity is needed to solve the problem at hand.
```